scripts/o3de/o3de/export_project.py

```diff
@@ -136,3 +207,322 @@ def add_args(subparsers) -> None:
+                                                               '--project-path', ctx.project_path,
+                                                               "--allowOverwrites",
+                                                               "--addDefaultSeedListFiles"]
+    ret = process_command(gen_engine_asset_list_command,
+                          cwd=ctx.engine_path if ctx.is_engine_centric else ctx.project_path)
+    if ret != 0:
+        raise RuntimeError(f"Error generating engine assets lists for {engine_asset_list_path}")
+
+    # Generate the bundles. We will place it in the project directory for now, since the files need to be copied multiple times (one for each separate launcher distribution)
+    gen_game_bundle_command = [asset_bundler_batch_path, "bundles",
+                                                         "--maxSize", str(max_bundle_size),
+                                                         "--platform", selected_platform,
+                                                         '--project-path', ctx.project_path,
+                                                         "--allowOverwrites",
+                                                         "--outputBundlePath", bundles_path / f"game_{selected_platform}.pak",
+                                                         "--assetListFile", game_asset_list_path]
+    ret = process_command(gen_game_bundle_command,
+                          cwd=ctx.engine_path if ctx.is_engine_centric else ctx.project_path)
+    if ret != 0:
+        raise RuntimeError(f"Error generating game bundle for {bundles_path / f'game_{selected_platform}.pak'}")
+
+    gen_engine_bundle_command = [asset_bundler_batch_path, "bundles",
+                                                           "--maxSize", str(max_bundle_size),
+                                                           "--platform", selected_platform,
+                                                           '--project-path', ctx.project_path,
+                                                           "--allowOverwrites",
+                                                           "--outputBundlePath", bundles_path / f"engine_{selected_platform}.pak",
+                                                           "--assetListFile", engine_asset_list_path]
+    ret = process_command(gen_engine_bundle_command,
+                          cwd=ctx.engine_path if ctx.is_engine_centric else ctx.project_path)
+    if ret != 0:
+        raise RuntimeError(f"Error generating engine bundle for {bundles_path / f'game_{selected_platform}.pak'}")
+
+    return bundles_path
+
+
+def kill_existing_processes(project_name: str):
+    o3de_process_names = ['o3de', 'editor', 'assetprocessor', f'{project_name.lower()}.serverlauncher',
+                          f'{project_name.lower()}.gamelauncher']
+    processes_to_kill = []
+    for process in psutil.process_iter():
+
```

Comment:
```suggestion
    # Don't try to close o3de.sh (the script currently running), but still look for o3de (Project Manager)
    if process.name() == 'o3de.sh'
        continue
```

Reply:
That check is already handled in the line below with the `O3DE_SCRIPT_NAME` global variable